### PR TITLE
closing issue 98, effectively removing section 5.4

### PR DESCRIPTION
**Pull request by fimbault**
- Reading the state of a request (#98) - removes section 5.4
- Updating and reading access tokens (#99) 
- Remove the links to already closed issues in the spec (<strike>issues #29, #55, #106</strike>, #98, #99)
- Update the list of high level changes for upcoming version 4 in appendix A

### CODE REVIEW

**Comment by yaronf** [comment]
Guys, please don't mix editorial and technical commits in the same PR. 😢

**Comment by jricher** [comment]
@yaronf this does not mix those types. This is a removal of a section that was marked for removal several months ago and has received no input to keep it, after several requests by the editors.

**Comment by jricher** [comment]
On second read, I realize I missed a change and I agree with Yaron: this PR should not remove issue links to unrelated issues. Those should be in a separate PR.

**Review by jricher — CHANGES_REQUESTED** [review]
This PR should only close the links for only the affected issues. A cleanup PR should be added separately as needed.

**Comment by fimbault** [comment]
Change done, only 98 and 99 are now concerned.

File: draft-ietf-gnap-core-protocol.md [hunk 1]
```diff
@@ -4301,6 +4273,7 @@ sure that it has the permission to do so.
 - -04
     - Updated terminology.
     - Refactored key presentation and binding.
+    - Closed issues related to reading and updating access tokens
```
Comment (jricher) [inline draft-ietf-gnap-core-protocol.md:4276]:
```suggestion
    - Removed function to read state of grant request by client
    - Closed issues related to reading and updating access tokens
```

**Review by jricher — CHANGES_REQUESTED** [review]
just needs the changelog updated (included) and otherwise looks good

File: draft-ietf-gnap-core-protocol.md (hunk 1 above)
Comment (fimbault) [reply draft-ietf-gnap-core-protocol.md:4277]: yes that's clearer, done

**Review by jricher — APPROVED** [review]

**Comment by davidgtonge** [comment]
I'm a bit late as this has already been approved. I'm ok with the text being removed, but I do think we need to address https://github.com/ietf-wg-gnap/gnap-core-protocol/issues/146

**Pull request merged**